### DISCUSSION TITLE
Option to select Audio output

### DISCUSSION THREAD

**Discussion by Rahjooyan** [ShankarBUS/ModernFlyouts#560] [Ideas]
hi, I'm using HDMI cable to watch online videos on TV but I have problem with Windows 10. when I click volume button on Taskbar to select HDMI Output, It takes some minutes.
Is it impossible to add feature to select Audio Output with this app?

Thanks.

**Comment by prakharb5** [comment]
Nothing ever is impossible :)

Regarding your request, please see #336. 

One more thing
Before opening any discussion, please search for your request in issues. Thanks

**Reply by Rahjooyan** [reply]
Thanks. I found requests here so I thought that I need to request feature in Discussions. I checked it and I didn't anything. sorry for my mistake.

**Reply by prakharb5** [reply]
No worries :D
